Resources/ServerInfo/_Moffstation/Guidebook/Mobs/Resomi.xml

```diff
@@ -0,0 +1,62 @@
+
+  Resomi packs would often name themselves after their pack, and will be more trusting and caring to members of their pack from formation to their eventual death. However, it has been known that some Resomi reject other Resomi when forming a packs, instead choosing to cling to items, ornaments or weapons for support and comfort.
+
+  While Resomi hatcheries has sought to maintain this culture, the Resomi's explansion into the across the stars have cause this historic culture to fade away. While some Resomi still chose to follow this structure, most forego it and go by other nicknames or names. With most names being named after colorful objects and food.
+
+  ## Nightvision
+  The Resomi have evolved to have light sensitive eyes which allows them to see in the dark. While most other species would require a flashlight or other light sourceto see in the dark,
+  Resomi can visually see that same as if they were in a dimly lit room.
+  However, headwear and glasses that with reflective coatings designed to protect the wearer from bright lights and flashes will block the Resomi's ability to see in the dark.
+
+  Any glasses that would provide protection from flashes will block their nightvision ability.
+  Using HUD variants allows you to keep your nightvision, at the cost of losing flash immunity.
+  
+  You can configure if you start with glasses or HUD in the loadouts for all relevant roles.
+  
+  ## Flash Vulnerability
+  Their increased sensitivity to light comes at a cost, with Resomi being highly vulnerable to bright flashes. While a handheld flash would normally disrupt someone's vision for [color=#ffe100]5 seconds[/color], Resomi's sensitivity to light causes a regular flash to last [color=#ff6f00]10 seconds.[/color]
+<Box>
+    <GuideEntityEmbed Entity="ClothingEyesGlassesSunglasses" Scale="4" Caption="Sunglasses block Nightvision"/>
+    <GuideEntityEmbed Entity="ClothingEyesHudCommand" Scale="4" Caption="HUDs will not block Nightvision"/>
+</Box>
+  ## HUD or Sunglasses
+  Wearing a HUD Interface grants Resomi to see in the dark. While Sunglasses and Welding Masks will block this ability. Resomi can choose to switch between these types of head and eye wear to best suit their current situation. However... Resomi who use to use their innate Nightvision, can see themselves being ambushed well prepared foes.
+
+  ## Damage Vulnerabilities
+  Due to their smaller statue, Resomi are vulnerable to all forms of physical [color=#ffae00]brute damage[/color]. Taking [color=#ff6f00]15% more brute damge[/color] than a regular human.
+The insulating nature of their feathers allow them to resist the cold, taking [color=#9ee5f7]10% less cold damage[/color]. While also being able to find comfort in cold and warmer climates.
```

Comment:
```suggestion
  Due to their smaller stature, Resomi are vulnerable to all forms of physical [color=#ffae00]brute damage[/color]. Taking [color=#ff6f00]15% more brute damage[/color] than a regular human.
However, the insulating nature of their feathers allow them to resist the cold, taking [color=#9ee5f7]10% less cold damage[/color]. While also being able to find comfort in both cold and warmer environs.
```
Climate -> Environ because a station would have environs, not climates.